tests/dispatchers/sync.test.ts

```diff
@@ -0,0 +1,179 @@
+import * as fs from 'fs/promises';
+import { handler } from '../../src/commands/dispatchers/sync';
+import * as dispatcherRepository from '../../src/repositories/dispatchers';
+import { generateMailAction, generateTaskAction } from '../__helpers__/actions';
+import { generateDispatcher, generateMinimalDispatcher } from '../__helpers__/dispatchers';
+
+describe('Dispatchers - Sync', () => {
+  let updateDispatcherSpy;
+  let updateActionSpy;
+  let deleteActionSpy;
+
+  const existingDispatcher = generateDispatcher();
+
+  beforeAll(() => {
+    // TODO: Move these somewhere else?
```

Comment:
Yes please, would be nice if we can make a dispatcherRepositoryMock file in the helpers 😄